src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -566,7 +1364,7 @@ def launch_fluent(
     processor_count: Optional[int] = None,
     journal_file_names: Union[None, str, list[str]] = None,
     start_timeout: int = 60,
-    additional_arguments: Optional[str] = None,
+    additional_arguments: Optional[str] = "",
```

Comment:
We were checking for `None` then setting it to `""`. By default it was `None` so we have changed it's default value from `None` to `""` and avoided `if` check.